chrisbanes/tivi
# Add open source licenses screen to Android app

current only enable on QaDebug flavor on android platform.

Test path:

1. execute ':android-app:app:generateLicensesAsset' task
2. run the project with android device.
3. go to the settings screen.
4. click the "View Open Source Licenses" item
5. check the "Open Source Licenses" page.

screenshot(video):

[![Watch the video](https://github.com/chrisbanes/tivi/assets/1454424/b42678eb-1279-4882-9e40-305a39fcb49c)](https://github.com/chrisbanes/tivi/assets/1454424/b42678eb-1279-4882-9e40-305a39fcb49c)



Fixes #1471
